RAT_README.txt

```diff
@@ -60,6 +60,7 @@ $MY_SQROOT/../sql/bin/SqlciErrors.txt
 
 dcs/conf/servers 
 dcs/conf/backup-masters 
+dcs/conf/master
                  :  Apache Trafodion DCS default configuration file
                  -> configuration file do not handle comments
 
```

Comment:
dcs/conf/master is another configuration file which contains the hostname.. It is  just like backup-masters file and as described in the comments it  does not handle comments